internal/controller/cloud-control/scope_gcp_test.go

```diff
@@ -60,6 +59,9 @@ var _ = Describe("Feature: KCP Scope", func() {
 				Should(Succeed(), "failed creating GardenerCluster CR")
 		})
 
+		// kymaNetwork is not ignored, and should reconcile into ready state with network ref in the status!!!
+		// a ready kymaNetwork is a prerequisite for Scope to become ready
+
```

Comment:
Could the required status fields be filled via `ObjActions` (`With...` ) instead?
What is the system under test here?